package_manifest.yaml

```diff
@@ -0,0 +1,383 @@
+---
+packages:
+  children:
+    foreman: {}
+    plugins: {}
+    katello: {}
+    katello-client: {}
+
+foreman:
+  vars:
+    package_base_dir: 'packages/foreman/'
+    releasers:
+    - koji-foreman-nightly
```

Comment:
I thought this releaser only worked with foreman, foreman-selinux, foreman-proxy and foreman-installer.

Reply:
This is essentially the "jenkins" releaser.

Reply:
Can obal handle using one releaser for a few packages while using another for the rest?

Reply:
Sure, just define them on the package instead of the group.

Reply:
What evgeni said.  The current releasers set in the PR at a group level were a best guess, and I am not familiar with the purpose of each releaser in foreman.  Therefore, if there are ones better suited for a group or package, please let me know.

Reply:
Essentially, this should be `koji-foreman` and either we set nightly package builds, e.g. foreman, to `koji-foreman-nightly` and then change that when we branch off to release branches OR we let the CI jobs handle overriding this when they need to. The nightly releasers that use Jenkins are not as simple as 'tito release' though so in general this particular releaser may require some thought or further configuration